The native Hot Rod client tests in 15.0.x are failing with GraalVM 25.

### Describe the bug

```
Caused by: java.lang.NullPointerException
2025-10-06T06:44:08.2593357Z        org.infinispan.client.hotrod.graalvm.substitutions.SubstituteAbstractInfinispanServerDriver.copyFromJar(SubstituteAbstractInfinispanServerDriver.java:27)
2025-10-06T06:44:08.2594757Z        org.infinispan.server.test.core.AbstractInfinispanServerDriver.copyProvidedServerConfigurationFile(AbstractInfinispanServerDriver.java:61)
2025-10-06T06:44:08.2595560Z        [...]
```

### Expected behavior

_No response_

### Actual behavior

_No response_

### How to Reproduce?

_No response_

### Output of `uname -a` or `ver`

_No response_

### Output of `java -version`

_No response_

### Infinispan version or git rev

_No response_

### Build tool (ie. output of `mvnw --version` or `gradlew --version`)

_No response_

### Additional information

_No response_

--- a/graalvm/client-hotrod/src/test/java/org/infinispan/client/hotrod/graalvm/substitutions/SubstituteAbstractInfinispanServerDriver.java
+++ b/graalvm/client-hotrod/src/test/java/org/infinispan/client/hotrod/graalvm/substitutions/SubstituteAbstractInfinispanServerDriver.java
@@ -24,7 +24,7 @@ import com.oracle.svm.core.annotate.TargetClass;
 public class SubstituteAbstractInfinispanServerDriver {
 
    public static void copyFromJar(String source, final Path target) throws URISyntaxException, IOException {
-      URI resource = SubstituteAbstractInfinispanServerDriver.class.getResource("").toURI();
+      URI resource = SubstituteAbstractInfinispanServerDriver.class.getResource("/").toURI();
       try (FileSystem fileSystem = FileSystems.newFileSystem(resource, Collections.emptyMap())) {
          final Path jarPath = fileSystem.getPath(source);
          Files.walkFileTree(jarPath, new SimpleFileVisitor<>() {

